docs/technical/security.md

```diff
@@ -0,0 +1,202 @@
+
+.NET (use by this project at time of writing) has code analysis tools, these should be enabled on all projects.
+
+Other languages have 'linting' tools which should be considered when onboarding a new language for this project.
+
+### Dependency Health
+
+#### .NET Packages
+
+.NET Core references published open source functionality via NuGet Packages.  These packages may have published security issues, Common Vulnerabilities and Exposures (CVE) or GitHub Security Advisory (GHSA) which will require investigation and replacement by the development team to ensure executing code has the best possible security posture.
+
+Tools like GitHub's [Dependabot](https://docs.github.com/en/code-security/dependabot) and [SonarQube](https://www.sonarqube.org/) have the capability to warn developers when these vulnerabilities are discovered.  
+Even the .NET command line tool provides a basic view of dependency issues:
+
+```cmd
+dotnet list package --vulnerable --include-transitive
+```
+
+This [tooling](https://devblogs.microsoft.com/nuget/how-to-scan-nuget-packages-for-security-vulnerabilities/) is very basic however.
+
+At a minimum tooling should be used to warn developers of issues would run in the build pipeline.  Ideally, developers are notified as soon as vulnerabilities are discovered and changes can be made even for components which aren't in active development.
+
+#### Password scanning
+
+All code repos in the Buerokratt platform are public on GitHub.  This means that secrets should be carefully segregated from code.  
+
+In order to prevent developers committing secrets to public repos this project should implement password scanning which would run before changes are committed.
+
+This can be done on a Git commit hook and is documented here:
+
+> See the CentOps repo for an example of this setup.  This should be applied to all repos.
+
+## Built Artefacts
+
+This project aims to package all shipping code in Docker Containers, therefore to remain as secure as possible the project will need to validate they are free of known vulnerabilities.
+
+### Containers
+
+Docker Containers most commonly use base images which contain various component which themselves may have security vulnerabilities.  Known CVEs may exist for components within these images which may expose vulnerabilities aside from the code and dependencies directly shipped by this project.
+
+Tools exist to scan containers for vulnerabilities.  One such tool is, Container Scan which can be run as part of the CI pipeline.
+
+[Container Scan](https://github.com/marketplace/actions/container-image-scan)
+
+Other container scanning technologies exist if this is found to be unsuitable.
+
+Some consideration should be given to containers which are in production when vulnerabilities are discovered.  This may mean rescanning containers on a regular basis and raising alerts if security issues are discovered.
+
+## Kubernetes
+

```

Comment:
Another thing that may be worth a shout is adding in how communication works with the control pane via API to manage the cluster e.g., create/delete resources via kubectl for example